hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/TeraGen.java

```diff
@@ -262,7 +262,8 @@ public void cleanup(Context context) {
   }
 
   private static void usage() throws IOException {
-    System.err.println("teragen <num rows> <dummy output dir> <pravega uri> <scope> <stream>");
+    System.err.println
```

Comment:
Can we use `slf4j` rather than `System.err` and `Syetem.out` for logging?

Reply:
Done